More fixing of lint errors for generated project

vmarta opened the pull request:
Related to #1264

vmarta committed:
More fixing of lint errors for generated project

kevinxh commented:
we have generated project CI tests, are we running `npm run lint` in those tests?

bfeister commented:
@vmarta - good idea from @kevinxh if we can get that in place

vmarta commented:
> we have generated project CI tests, are we running `npm run lint` in those tests?

@kevinxh we don't run it in CI. All right, let me see about getting it to run the linting.

vmarta committed:
Does the new linting test fail in CI?

vmarta committed:
Fix more linting errors

vmarta commented:
@kevinxh @bfeister Ok, I've added the CI test to lint the generated project. And also fixed more linting errors.

Please review the PR again, thanks!

vmarta commented on .github/workflows/test.yml:
```diff
@@ -277,6 +283,12 @@ jobs:
           GENERATOR_PRESET: ${{ matrix.template }}
         timeout-minutes: 7
 
+      - name: Lint the generated project
+        if: env.IS_TEMPLATE_FROM_RETAIL_REACT_APP == 'true'
+        uses: "./.github/actions/linting"
+        with:
+          cwd: ${{ env.PROJECT_DIR }}
```
One test for `generated` and the other for `generated-windows`.

vmarta committed:
Merge branch 'develop' into handlebars-formatting
